### PR TITLE
add a polytope method to hypergeometric motives

### PR DESCRIPTION
**Pull request by fchapoton**
This is adding one new method to the class of hypergeometric motives, namely the construction of a lattice polytope.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [x] I have created tests covering the changes.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/modular/hypergeometric_motive.py [hunk 1]
```diff
@@ -1131,6 +1137,44 @@ def canonical_scheme(self, t=None):
         ideal = ring.ideal([eq0, eq1, self.M_value() * eq2_neg - t * eq2_pos])
         return Spec(ring.quotient(ideal))
 
+    def polytope(self):
```
Comment (mkoeppe) [inline src/sage/modular/hypergeometric_motive.py]: I would suggest to call it `lattice_polytope`
and provide a method `polyhedron` that returns a Polyhedron

Comment (fchapoton) [reply src/sage/modular/hypergeometric_motive.py]: I prefer to have just one method.

Comment (mkoeppe) [reply src/sage/modular/hypergeometric_motive.py]: Makes sense, I don't question your motives ;)

**Comment by fchapoton** [comment]
good to go ?

**Review by mkoeppe — APPROVED** [review]
LGTM

**Comment by fchapoton** [comment]
thanks

**Pull request merged**